src/components/Reports/GDPRPermissions/PermissionExpandableRow.js

```diff
@@ -118,6 +118,14 @@ const PermissionExpandableRow = ({ field, email, dependencies: { dopplerApiClien
                         <th aria-label={_('subscriber_gdpr.modification_date')} scope="col">
                           <FormattedMessage id="subscriber_gdpr.modification_date" tagName="span" />
                         </th>
+                        {permissions.some((p) => p.originType) ? (
```

Comment:
Shouldn't this column has to show always beyond having this data or not?  Is there a reason to have a permission without origin? In this case, maybe it could shows a text like `none`/`vacío`

Reply:
If this is something that we don't have from the BE yet, I think that this is ok like that.

Reply:
We don`t have yet this information from API so, the column should only be shown when the data is ready.

Reply:
Thanks for the clarification!